### DISCUSSION TITLE
Is it because the recent update the mod isnt working?

### DISCUSSION THREAD

**Discussion by LyomaBeans** [thatskymod/Sky-CotL-Scripts#116] [Q&A] [answered]
I go to use mod and it just gets stuck at a percentage or says error when trying to do dailies or send light D: worst time for it to not work LMAOOO

**Comment by Sbst397** [comment]
You too?? For me all the mod doesn't work and Canvas crash constantly it's unbearable nothing works anymore

**Reply by LyomaBeans** [reply]
yeah i tried loading it with both mods and it would just crash but then i tried loading it up with mods separately and it was fine but when running the commands it would just say error

**Reply by Sbst397** [reply]
Exactly the same for me but it seems they're not doing any updates, It's really disadvantageous, knowing a ts group will come soon.

**Comment by thatskymod** [comment, marked as answer]
Hello,

I apologize for the outage. That said, any time the base game is updated, mods also need to be aligned with the new logic. This may take some time.

Consider trying the newly released [libLangit046.so](https://github.com/thatskymod/Sky-CotL-Scripts/commit/21df5cdfb0eb75710bf49820ee1e234e2ad26abd). This version should resolve any outstanding issues found in its predecessors.
